Sources/TuistSupport/Utils/TemporaryDirectory.swift

```diff
@@ -40,7 +39,7 @@ public final class TemporaryDirectory {
         // which will be the actual path to the temporary directory.
         var template = [UInt8](path.pathString.utf8).map { Int8($0) } + [Int8(0)]
 
-        if TSCLibc.mkdtemp(&template) == nil {
```

Comment:
As far as I understand TSCLibc just import Darwin and use it. So we don't need to import it

Reply:
well, it looks like no. Do you have any ideas @fortmarek

Reply:
I believe for Darwin, we need to add an exception in the logic for checking implicit imports as that framework is implicitly present on Darwin platforms.

Reply:
@fortmarek It works right now. Now not implicit dependencies were found and all tests are passed. Lets merge?